demo/apps/todomvc/basis/task.js

```diff
@@ -0,0 +1,57 @@
+var entity = require('basis.entity');
+var Filter = require('basis.data.dataset').Filter;
+
+var Task = entity.createType('Task', { name: String, completed: Boolean, created: Number });
+var LS_KEY = 'basisjs-todos';
+
+Task.extendClass(function(_super, proto){
+  return {
+    init: function(){
+      proto.init.apply(this, arguments);
```

Comment:
Looks redundant

Reply:
seems like I forgot to remove this after some experiments...